### PR TITLE
Elixir and PHP now use Treesitter grammars

### PR DESCRIPTION
**Pull request by lildude**
## Description
GitHub now uses the Treesitter grammars for Elixir and PHP. This PR updates the `script/list-grammars` script and the resulting README.md to reflect this.

_[ Template removed as it doesn't apply ]_

### CODE REVIEW

File: script/list-grammars [hunk 1]
```diff
@@ -9,10 +9,12 @@ TREE_SITTER_GRAMMARS = {
   "CodeQL" => "https://github.com/tree-sitter/tree-sitter-ql",
   "CSS" => "https://github.com/tree-sitter/tree-sitter-css",
   "EJS" => "https://github.com/tree-sitter/tree-sitter-embedded-template",
+  "Elixir" => "https://github.com/elixir-lang/tree-sitter-elixir",
```
[the remark this answers is not in the document]
Comment (lildude) [reply script/list-grammars:12]: I'm not entirely sure as I'm not part of that team. I'll see if I can find out.

[the remark this answers is not in the document]
Comment (plusgut) [reply script/list-grammars:12]: @lildude were you able to find out? I'm also very curious about tree-sitter grammars from the community

[the remark this answers is not in the document]
Comment (lildude) [reply script/list-grammars:12]: Yes. See https://github.com/github-linguist/linguist/discussions/6073

[the remark this answers is not in the document]
Comment (plusgut) [reply script/list-grammars:12]: Thank you very much for answering so quickly.

**Pull request merged**